specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -18234,6 +19245,16 @@
       },
       "additionalProperties": false
     },
+    "ModelSettings": {
+      "additionalProperties": false,
+      "properties": {
+        "modelId": {
```

Comment:
>modelId [](http://example.com/codeflow?start=9&length=7)

Description? Include information about what type/format this ID is.

Reply:
updated